### ISSUE TITLE
Not allowed to load local resource:

### ISSUE THREAD

**Issue by sabarinathancs** [paulsutherland/Polyonic#50]
While Try to Force Reload . I was getting this error ,and getting is full blank screen

**Comment by paulsutherland** [comment]
@sabarinathancs Hi - what platform is this for?

**Comment by sabarinathancs** [comment]
Windows

**Comment by paulsutherland** [comment]
@sabarinathancs 

Can you give me a bit more information to go on so I can try and recreate the issue.

What platform were you targeting? Mobile, Web or Desktop (Electron)?
What command were you running? npm run electron:dev npm run ionic? ...
Force reload - is that the browser, code editor?

If you can outline the steps you took I can review it.

Thanks,

Paul

**Comment by sabarinathancs** [comment]
yup ok, let me explain. my target paltform is windows and android, to run code i’m using npm rin electron:local. when problem is coming is after app loaded in electron i’m trying too use cmd+r
after that entire application got white screen and error through  in console Not allowed to load local resource:

**Comment by paulsutherland** [comment]
@sabarinathancs Ok thanks.  I can recreate this issue.  It is related to angular and routing.  It tries to load a route view on force reload and it can't obviously find that in the the www directory.  I will need to work on this to get it reload the original index page on reload or force reload.

I'll push an update out when I fix it.

Thanks,

Paul

**Comment by paulsutherland** [comment]
@sabarinathancs I have pushed a fix.  Please try it out and let me know if it clears the issue for you.
